spl/Cargo.toml

```
@@ -23,6 +23,7 @@ token_2022 = ["spl-token-2022"]
 [dependencies]
 anchor-lang = { path = "../lang", version = "0.28.0", features = ["derive"] }
 borsh = { version = ">=0.9, <0.11", optional = true }
+mpl-token-auth-rules = "1.4.3"
```

It should be optional:
```suggestion
mpl-token-auth-rules = { version = "1.4.3", optional = true }
```